payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayLauncherActivity.kt

```diff
@@ -132,12 +129,10 @@ internal class GooglePayLauncherActivity : AppCompatActivity() {
                 }
             }
         } else {
-            lifecycleScope.launch {
```

Comment:
I also moved other suspend fun calls from the activity to the view model, but this was the easy part.